quadriga/src/main/java/edu/asu/diging/quadriga/api/v1/AddNetworkApiController.java

```diff
@@ -59,31 +59,43 @@
     @ResponseBody
     @RequestMapping(value = "/api/v1/collection/{collectionId}/network/add", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
     public HttpStatus processJson(@RequestBody Quadruple quadruple, @PathVariable String collectionId) {
+
+        // First we check whether a quadruple is present in request body
+        if (quadruple == null) {
+            logger.error("Quadruple not present in network submission request for collectionId: "  + collectionId);
+            return HttpStatus.NO_CONTENT;
```

Comment:
this should return a bad request. No content means:
> The server has fulfilled the request but does not need to return an entity-body, and might want to return updated metainformation. The response MAY include new or updated metainformation in the form of entity-headers, which if present SHOULD be associated with the requested variant. [...]

https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html